brewtils/plugin.py

```diff
@@ -28,10 +29,8 @@
 # This is what enables request nesting to work easily
 request_context = threading.local()
 
-# These are not thread-locals - they should be set in the Plugin __init__ and then never
-# touched. This allows us to do sanity checks when creating nested Requests.
-_HOST = ""
-_PORT = None
+# Global config, used to simplify BG client creation and sanity checks.
+CONFIG = Box(default_box=True)
```

Comment:
I think this should be a private variable.

Reply:
The issue there is that the whole point of this is to be accessed from other modules (eg. `SystemClient`)